typing/typemod.ml

```diff
@@ -989,9 +989,9 @@ and transl_recmodule_modtypes env sdecls =
     List.fold_left
       (fun env (id, _, mty) -> Env.add_module ~arg:true id mty env)
       env curr in
-  let make_env2 curr =
+  let make_env2 ?(arg=true) curr =
```

Comment:
Cosmetic nitpick: there are four uses of this function, two with arg=true, two with arg=false; the benefit of making the argument optional is unclear.  Being explicit on two more call sites seems better.

Reply:
OK, since the uses are only inside the same function, I switched to using a non-defaulted argument.